src/System.Net.Sockets/tests/FunctionalTests/System.Net.Sockets.Tests.csproj

```diff
@@ -64,7 +64,15 @@
       <Link>Common\System\Net\Capability.Sockets.cs</Link>
     </Compile>
   </ItemGroup>
-
+  
+    <!-- Windows only tests -->
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="SocketOptionNameTest.cs" />
```

Comment:
+1. I missed this in my first look. I don't expect any issues--the shims should take care of rejecting this option--but we should definitely have a test to ensure that behavior.